vprof/code_heatmap.py

```diff
@@ -124,8 +125,7 @@ def run_as_package_path(self):
     def run_as_module(self):
         """Runs program as module."""
         try:
-            with open(self._run_object, 'r') as srcfile,\
-                _CodeHeatmapCalculator() as prof:
+            with open(self._run_object, 'r') as srcfile, _CodeHeatmapCalculator() as prof:
```

Comment:
I did that since there were few other lines with char size > 80. So I thought maybe the limit here is 100 char.